platform-operator/README.md

```diff
@@ -386,5 +405,39 @@ The default dev pipeline runs these Tekton steps:
 
 This system lets you use simple development builds while you're coding, then can automatically get more rigorous security and quality checks when you're ready to deploy to production environments.
 
-You can find the sample steps and template here:  https://raw.githubusercontent.com/kagenti/kagenti-operator/main/platform-operator/config/samples/tekton
+You can find the sample steps and template here:  <https://raw.githubusercontent.com/kagenti/kagenti-operator/main/platform-operator/config/samples/tekton>
+
+## Operator Development and Testing
+
+1. Fork the repository and create a new branch
+1. Modify the operator code to fit your needs
+   * e.g. the code for managing the deployed agents and tools is [./internal/deployer/kubernetes/kubernetes.go](./internal/deployer/kubernetes/kubernetes.go) file
+1. [Install Kagenti](https://github.com/kagenti/kagenti/blob/v0.0.4-alpha.18/docs/demos.md#installation)
+1. Uninstall the `kagenti-operator`:
+
+   * ```console
+     helm uninstall kagenti-platform-operator -n kagenti-system
+     ```
+
+1. * Sometimes this removes operator crds. When you get errors about missing crds,
+     just install them manually with: `kubectl create -f config/crds/bases/`
+
+1. switch to your new operator branch
+1. execute operator cleanup, build, and deployment:
```

Comment:
```suggestion
1. Sometimes this removes operator crds. When you get errors about missing crds, just install them manually with: `kubectl create -f config/crds/bases/`
1. Switch to your new operator branch
1. Execute operator cleanup, build, and deployment:
```

Reply:
The markdown linter did this. It's the correct format